### PR TITLE
doc(RHTAPWATCH-1007): demo workflow

### LINKED ISSUE #43: Document one opinionated flow from creating the cluster to onboarding a user and running a pipeline with webhook

### LINKED ISSUE #47: Check if smee-client can be deployed with less memory/cpu
1. The client was deployed with 32Mi and later moved to 320Mi, to troubleshoot some issues, but it might be that it's not required and memory amount can be reduced.
2. The client is configured to use 1 cpu, which is probably not required. We should try reducing the amount.

### LINKED ISSUE #48: README: Missing requirements
- podman/docker
- kubectl
- git

### PR DESCRIPTION
**Pull request by yftacherzog**
Document a demo workflow intended to be as streamlined as possible.

Closes #48 
Closes #47 
Closes #43

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,42 +67,259 @@ and modify the `system-reserved` line under `kubeletExtraArgs`:
...
+   suffix, while the `Repository` URL doesn't.
+
+   Deploy the manifests:
+
+```bash
+kubectl create -f ./test/resources/demo-users/user/ns2/application-and-component.yaml
+```
+
+**Note:** Further explanation about those resources can be found in
+[later section](#create-an-application-and-component).
```
Comment (avi-biton) [inline README.md]: This link opens the README again

Comment (yftacherzog) [reply README.md]: It should point to a section in the README. Let me have a look.

Comment (yftacherzog) [reply README.md]: I think I renamed the section and broke the link. Hang on.

Comment (yftacherzog) [reply README.md]: Should be ok now

Comment (avi-biton) [reply README.md]: Yes, it is.
Thanks :)

**Review by avi-biton — APPROVED** [review]
Looks great

**Comment by yftacherzog** [comment]
I'll wait for https://github.com/konflux-ci/testrepo/pull/22 to be merged before merging this one as it relies on the content of that PR.

**Review by gbenhaim — APPROVED** [review]
looks great !

**Pull request merged**